infra/modules/terraform-backend-s3/main.tf

```
@@ -47,6 +47,8 @@ resource "aws_s3_bucket" "tf_state" {
   bucket = local.tf_state_bucket_name
 
   # checkov:skip=CKV_AWS_144:Cross region replication not required by default
+  # checkov:skip=CKV2_AWS_62:S3 bucket does not need notifications enabled
+  # checkov:skip=CKV2_AWS_61:This s3 bucket holds state files that we do not need to delete
```

maybe a more description reason for ignoring like

```suggestion
  # checkov:skip=CKV2_AWS_61:No need to define S3 bucket lifecycle configuration to expire or transition tfstate files since they will always be needed and the file sizes are small anyways
```

based on our discussion over slack